cypress/integration/Manual Listing Pages/ManualListingPages.feature

```diff
@@ -0,0 +1,157 @@
+        And the page "1" is highlighted
+        And each result displays the trial title with a link in the following format "/clinicaltrials/NCI-"
+        And each result displays the trial summary
+        And each result displays "Location: " below the summary
+        And the 2 pager is displayed with the following "<pager items>"
+
+        Examples:
+            | screen breakpoint | path                                                   | page title                                                             | pager items  |
+            | desktop           | /about-cancer/treatment/clinical-trials/covid-19       | NCI-Supported Clinical Trials for Coronavirus                          | 1,2,Next > |
+            | mobile           | /about-cancer/treatment/clinical-trials/covid-19       | NCI-Supported Clinical Trials for Coronavirus                          | 1,2,Next > |
+            | desktop           | /about-cancer/treatment/clinical-trials/cam-procedures | Clinical Trials for Complementary or Alternative Medicine Procedure(s) | 1,2,3,Next > |
+            | mobile            | /about-cancer/treatment/clinical-trials/cam-procedures | Clinical Trials for Complementary or Alternative Medicine Procedure(s) | 1,2,3,Next > |
+            | desktop           | /about-cancer/treatment/clinical-trials/adult-metastatic-brain-tumors | Clinical Trials to Treat Adult Metastatic Brain Tumors | 1,2,3,4,5,Next > |
+            | mobile            | /about-cancer/treatment/clinical-trials/adult-metastatic-brain-tumors | Clinical Trials to Treat Adult Metastatic Brain Tumors | 1,2,3,4,5,Next > |
+            | desktop           | /about-cancer/treatment/clinical-trials/kidney-cancer | Clinical Trials to Treat Kidney (Renal Cell) Cancer | 1,2,3,...,7,Next > |
+            | mobile            | /about-cancer/treatment/clinical-trials/kidney-cancer | Clinical Trials to Treat Kidney (Renal Cell) Cancer | 1,2,3,...,7,Next > |
+            | desktop           | /about-cancer/treatment/clinical-trials/transitional-cell-cancer-renal-pelvis-ureter | Clinical Trials for Transitional Cell Cancer of the Renal Pelvis and Ureter | 1,2,Next > |
+            | mobile            | /about-cancer/treatment/clinical-trials/transitional-cell-cancer-renal-pelvis-ureter | Clinical Trials for Transitional Cell Cancer of the Renal Pelvis and Ureter | 1,2,Next > |
+
+
+    Scenario Outline: All manual trial listing page components are displayed (without pagers)
+        Given screen breakpoint is set to "<screen breakpoint>"
+        When user is navigating to "<path>"
+        Then page title is "<page title>"
+        And the page displays pager info "Trials 1-N of N"
+        And the pager is not displayed
+        And each result displays the trial title with a link in the following format "/clinicaltrials/NCI-"
+        And each result displays the trial summary
+        And each result displays 'Location: ' below the summary
+        Examples:
```

Comment:
same here - add the step for other pages on line 38
'And delighter is displayed'